cacheops/conf.py

```diff
@@ -32,7 +33,7 @@ class Defaults:
 class Settings(object):
     def __getattr__(self, name):
         res = getattr(base_settings, name, getattr(Defaults, name))
-        if name == 'CACHEOPS_PREFIX':
+        if name in ['CACHEOPS_PREFIX', 'CACHEOPS_SERIALIZER']:
             res = res if callable(res) else import_string(res)
```

Comment:
Resolve if string, leave as is in other case. Will allow to use a module, a class or an instance.